### DISCUSSION TITLE
specify the operation after selecting the text

### DISCUSSION THREAD

**Discussion by jackjin-cn** [plateaukao/einkbro#294] [Ideas]
Can you specify the operation after selecting the text? For example, the current process is that after selecting the text, you need to select the corresponding operation or app. Can you select the word and directly enter the dictionary to look up the word, which reduces one step? When I read English web pages, I often need to look up words.

**Comment by plateaukao** [comment]
@jackjin-cn 
will consider adding this feature. Thanks for the suggestion.
